### ISSUE TITLE
NoMethodError: undefined method `colorize' for true:TrueClass

### ISSUE THREAD

**Issue by rnystrom** [planetscale/planetscale_rails#5]
Heyo! I'm giving this gem a shot and am running into this error locally:

```
NoMethodError: undefined method `colorize' for true:TrueClass
    raise "You must have `pscale` installed on your computer" unless command?("pscale").colorize(:red)
```

- I have pscale v 0.130.0 installed via homebrew at `/usr/local/homebrew/bin/pscale`
- I successfully created a demo branch via `pscale branch switch rn-test-branch --database campsite-api --create`
- I'm running `bundle exec rails psdb:migrate` when the error occurs
- I get the same error running `bundle exec rails psdb:setup_pscale`
- I installed the gem via

```rb
group :development do
  gem 'planetscale_rails'
end
```

One note is that I didn't have a `.pscale.yml`. The script generated one and put it in our project root which isn't actually our API root. I was hitting an error that this file didn't exist, moved it to our API project root, and that error went away.

**Comment by mscoutermarsh** [comment]
hey @rnystrom!

Opened a PR to fix one of the bugs, will get that released on rubygems soon and update when it is.

The other problem looks like the rake task can't detect that you have `pscale` installed.

Could you run `which pscale`? (that's what the rake task does to check)

Does it find the correct path?

**Comment by mscoutermarsh** [comment]
Ignore the last part, I think my PR will fix it. 😄

**Comment by mscoutermarsh** [comment]
Just published [0.1.3](https://rubygems.org/gems/planetscale_rails/versions/0.1.3) with the fix!

**Comment by rnystrom** [comment]
You rock, thanks!
